packages/web/src/brandkit/tracking.ts

```diff
@@ -0,0 +1,22 @@
+import analytics from 'src/analytics/analytics'
```

Comment:
Seems like this won't be just a brandkit requirement right? Maybe this should be further generalized

Reply:
thats a good point! although the trackDownload function itself is really just for brandkit.